src/utils/tree_node.C

```diff
@@ -527,22 +528,14 @@ const Elem* TreeNode<N>::find_element_in_children(const Point& p, const std::set
   // the point p.  So, let's look at the other children
   // but exclude the one we have already searched.
   for (unsigned int c=0; c<children.size(); c++)
-    if (c != excluded_child)
+    if (!searched_child[c])
       {
-        if (children[c]->active())
-          {
-            const Elem* e = children[c]->find_element(p,allowed_subdomains);
+        const Elem* e =
+          children[c]->find_element(p,allowed_subdomains,
```

Comment:
This is the call that requires that https://github.com/libMesh/libmesh/pull/451/files?diff=split#diff-31ecb7de88a89c206400bd14cf60de8fR473 NOT be there.

The only way we got here is if the child failed the bounding box check... so if we check it inside find_element() then we're not doing ANYTHING at all here...